Max/src/main/java/axi/Broker.java

```diff
@@ -0,0 +1,114 @@
+package axi;
+
+import axi.excepciones.InstrumentoDuplicadoException;
+import axi.excepciones.InstrumentoNoEncontradoException;
+import axi.modelos.InstrumentoFinanciero;
+import axi.modelos.InstrumentoFinancieroFactory;
+import axi.modelos.Inversor;
+import axi.modelos.Tipo;
+
+import java.util.ArrayList;
+
+public class Broker {
+    final static String NAME = "AXI";
+    private static Broker broker;
+    private String name;
+    private ArrayList<InstrumentoFinanciero> instrumentos;
+    private ArrayList<Inversor>inversores;
+
+    private Broker() {
+        this.name = NAME;
+        this.instrumentos = new ArrayList<>();
+        this.inversores = new ArrayList<>();
+    }
```

Comment:
Hay 2 propiedades que podrian ser una misma, tenemos una constante que se asigna a otra propuedad, ambas de la misma clase, podemos juntar estas dos en una misma

```suggestion
    private static Broker broker;
    private String name = "AXI";
    private ArrayList<InstrumentoFinanciero> instrumentos;
    private ArrayList<Inversor>inversores;

    private Broker() {
        this.instrumentos = new ArrayList<>();
        this.inversores = new ArrayList<>();
    }
```